### PR TITLE
Great big bug-fix PR (fix styles)

### LINKED ISSUE #119: Inconsistent button behaviour on touch devices
On touch devices the button stays in the "hover"-state when tapping.

### LINKED ISSUE #120: Checkbox on iOS has rounded corners
Should be square.

### LINKED ISSUE #121: Checkbox has outline
The checbox has a custom focus outline, but the default one is stil there.

### LINKED ISSUE #123: Bullet list css is missing in portal page with all components

### PR DESCRIPTION
**Pull request by piofinn**
## 📥 Proposed changes

Fix style-related issues on several components:
* Buttons no longer behave unnaturally on touch
* Checkbox no longer has outline when using mouse
* Checkbox no longer has rounded corners on iOS safari
* TertiaryButton and Accordion no longer turn white when pressed in Safari
* BulletList now has correct style in the Portal

## ☑️ Submission checklist

-   [x] I have read the [CONTRIBUTING]() doc
-   [x] `yarn build` works locally with my changes
-   [x] I have added tests that prove my fix is effective or that my feature works
-   [x] I have added necessary documentation (if appropriate)

## 💬 Further comments

Fixes #119, fixes #120, fixes #121, fixes #123

### CODE REVIEW

File: packages/button/button.scss [hunk 1]
```diff
@@ -30,11 +30,17 @@ $hover-elevation-distance: -0.25rem;
     transition: transform $animation-timing;
     position: relative;
 
-    &:focus,
+    &:-moz-focusring {
+        // get rid of dotted border on Firefox
+        outline: none;
+    }
+
+    html:not([data-mousenavigation]) &:focus,
```
Comment (Saegrov) [inline packages/button/button.scss]: Have you spoken with the designers about this? The focused-state does not have any scaling in Figma and this is a design change.

File: packages/checkbox/checkbox.scss [hunk 2]
```diff
@@ -18,8 +18,10 @@ $checkbox-focus-outline-color: #509ce6;
         -webkit-appearance: none;
         background-color: $checkbox-unchecked-background-color;
         border: solid rem(1px) $checkbox-border-color;
+        border-radius: 0;
```
Comment (Saegrov) [inline packages/checkbox/checkbox.scss]: Should add a comment that this fixes iOS Safari rounded corners-.

File: packages/accordion/accordion.scss [hunk 3]
```diff
@@ -36,6 +36,7 @@ $chevron-weight--hover: 0.1875rem;
         outline: none;
         font-family: inherit;
         background-color: inherit;
+        color: inherit;
```
Comment (Saegrov) [inline packages/accordion/accordion.scss]: Maybe add a comment on why this is here?

File: packages/accordion/accordion.scss [hunk 4]
```diff
@@ -63,6 +64,7 @@ $chevron-weight--hover: 0.1875rem;
         height: auto;
         padding: $content-vertical-padding $horizontal-padding;
         padding-top: $content-vertical-padding - $title-vertical-padding;
+        color: inherit;
```
Comment (Saegrov) [inline packages/accordion/accordion.scss]: Same as above.

File: packages/button/button.scss [hunk 5]
```diff
@@ -30,11 +30,17 @@ $hover-elevation-distance: -0.25rem;
     transition: transform $animation-timing;
     position: relative;
 
-    &:focus,
+    &:-moz-focusring {
+        // get rid of dotted border on Firefox
```
Comment (Saegrov) [inline packages/button/button.scss]: :+1:

File: packages/button/button.scss (hunk 1 above)
Comment (piofinn) [reply packages/button/button.scss]: Good point! Checked, and was OK 👍

**Pull request merged**